src/Modules/UngradedModules/FreeMod.jl

```diff
@@ -22,6 +22,22 @@ function FreeMod(R::AdmissibleModuleFPRing, names::Vector{Symbol}; cached::Bool=
   return FreeMod{elem_type(R)}(length(names), R, names)
 end
 
+function FreeMod(R::PBWAlgQuo, n::Int, name::VarName = :e; cached::Bool = false) # TODO cached?
+  return FreeMod{elem_type(R)}(n, R, [Symbol("$name[$i]") for i=1:n])
+end
+
+function FreeMod(R::PBWAlgRing, n::Int, name::VarName = :e; cached::Bool = false) # TODO cached?
+  return FreeMod{elem_type(R)}(n, R, [Symbol("$name[$i]") for i=1:n])
+end
+
+function FreeMod(R::PBWAlgQuo, names::Vector{String}; cached::Bool=false)
```

Comment:
```suggestion
function FreeMod(R::PBWAlgQuo, names::Vector{<:VarName}; cached::Bool=false)
```
To allow more types of inputs